src/nat/front_ends/fastapi/job_store.py

```
@@ -83,101 +263,336 @@ def create_job(self,
+            serialized to JSON for storage.
+
+        Raises
+        ------
+        ValueError
+            If the specified job_id does not exist in the job store.
+        """
+
+        async with self.session() as session:
+            job: JobInfo = await session.get(JobInfo, job_id)
+            if job is None:
+                raise ValueError(f"Job {job_id} not found in job store")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)